wanted-next-generation-software-engineer.md

```diff
@@ -2,27 +2,23 @@
 
 Hi There,
 
-Our company AutoScout24 GmbH (http://www.autoscout24.com/) is searching for Senior Engineer with relocation to Munich,
+Our company AutoScout24 GmbH (http://www.autoscout24.com/) is searching for a Senior Engineer with relocation to Munich,
```

Comment:
another english speaking colleagues actually insist to remove `a` here, since if you put `a` it means we searching for only one Senior Engineer :)